pkg/trait/kamelets.go

```diff
@@ -199,39 +184,8 @@ func (t *kameletsTrait) addKamelets(e *Environment) error {
 	return nil
 }
 
-func (t *kameletsTrait) configureApplicationProperties(e *Environment) error {
```

Comment:
why is this being removed? we need to keep setting the Kamelet default property values on the application properties or how is this going to be done?

Reply:
They are already managed by the runtime in #4797